adal/src/main/java/com/microsoft/aad/adal/AuthenticationActivity.java

```diff
@@ -156,10 +156,11 @@ protected void onCreate(Bundle savedInstanceState) {
         CookieManager cookieManager = CookieManager.getInstance();
         cookieManager.setAcceptCookie(true);
 
+        Logger.v(TAG + methodName, "AuthenticationActivity was created.");
         // Get the message from the intent
         mAuthRequest = getAuthenticationRequestFromIntent(getIntent());
         if (mAuthRequest == null) {
-            Log.d(TAG, "Request item is null, so it returns to caller");
+            Log.d(TAG, "Intent for Authentication Activity doesn't have the request details, returning to caller");
```

Comment:
I'm not planning to do that for public ADAL. I alreay heard that it's causing parsing problem for client apps, and i don't think it's a good practice to include method name in the TAG.